connexta/eve
# Patch: Media Component, Release Version not working with Eve filepath

#### What does this PR do?

The media and release version components are currently broken on the DEV branch.  I believe the problem is that since Bates made the `eve` folder persistent, our changes are not deployed to the file path (which is as it should be).  The problem is that the necessary files the media component/release version component needs were not initialized, this PR should fix that.  If the `carousel.json` file or `carouselMedia` directory are not present in the `eve` folder, they are created.

#### Who is reviewing it?

@vinamartin @alchucam @mcalcote @haydenudelson 

#### How should this be tested?

Make sure the `eve` folder is empty, then run `yarn go` and `yarn server` or `make go` and add/remove media in the media component.  It should work without any errors.  Edit version numbers in the GSR component, this should also work.

#### Screenshots

#### Checklist:

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
